### PR TITLE
Added missing + unused entities and recreated SuppresCrosshair

### PR DESCRIPTION
**Pull request by FriskTheFallenHuman**
This PR adds the following entities listed on valve developer wiki, this entities are safe to add since the code is provided on the base code:

- [func_bulletshield](https://developer.valvesoftware.com/wiki/Func_bulletshield)
- [trigger_super_armor](https://developer.valvesoftware.com/wiki/Trigger_super_armor)
- [trigger_portal](https://developer.valvesoftware.com/wiki/Trigger_portal)
- [weight_button](https://developer.valvesoftware.com/wiki/Func_weight_button)

this pr also updates the .bat files and also includes SuppresCrosshair From Portal 1 recreated

### CODE REVIEW

**Comment by Blixibon** [comment]
I'm trying to lock in Mapbase v5.0's features since this update is getting too large and experimental for the desired circumstances.

I need to spend more time preparing and testing the already upcoming changes, so I'm not going to be able to review this until after Mapbase v5.0 is released. I'm sorry if you wanted your changes to be included in Mapbase v5.0, but it can still come in a later update. I'll leave this PR open until that time comes.

File: sp/src/createallprojects.bat [hunk 1]
```diff
@@ -1 +1,4 @@
-devtools\bin\vpc.exe /hl2 /episodic +everything /mksln everything.sln
+@echo off
+cls
+	devtools\bin\vpc.exe /hl2 /episodic +everything /mksln everything.sln
```
Comment (Blixibon) [inline sp/src/createallprojects.bat:3]: What is the purpose of these batch file changes?

File: sp/src/game/client/mapbase/c_trigger_portal.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//========= Copyright Valve Corporation, All rights reserved. ============//
```
Comment (Blixibon) [inline sp/src/game/client/mapbase/c_trigger_portal.cpp:1]: As part of the v5.0 update's new contribution guidelines, this should now use the Mapbase header.

File: sp/src/game/server/hl2/hl2_player.cpp [hunk 3]
```diff
@@ -241,10 +241,9 @@ class CLogicPlayerProxy : public CLogicalEntity
 	void InputEnableCappedPhysicsDamage( inputdata_t &inputdata );
 	void InputDisableCappedPhysicsDamage( inputdata_t &inputdata );
 	void InputSetLocatorTargetEntity( inputdata_t &inputdata );
-#ifdef PORTAL
```
Comment (Blixibon) [inline sp/src/game/server/hl2/hl2_player.cpp:244]: I don't think removing all evidence that this was originally from Portal is wise, but I can add comments about it myself later.

**Review by Blixibon — COMMENTED** [review]
I was finally able to properly review this PR.

I left a few comments about things I was concerned about; This PR may need to be updated before it can be merged.

File: sp/src/createallprojects.bat (hunk 1 above)
Comment (FriskTheFallenHuman) [reply sp/src/createallprojects.bat:3]: oops this should not had to be here :sweat_smile:

File: sp/src/game/client/mapbase/c_trigger_portal.cpp [hunk 4]
```diff
@@ -1,4 +1,4 @@
-//========= Copyright Valve Corporation, All rights reserved. ============//
+//================== Mapbase - Source 2013, Author(s) Krispy ===================//
```
Comment (Blixibon) [inline sp/src/game/client/mapbase/c_trigger_portal.cpp:1]: This is the wrong header formatting. "Mapbase - Source 2013" should be followed by the link to the repo while the "Author(s)" part should be below "Purpose:".

For example, see how I formatted the headers in reductor's VScript implementation files:
https://github.com/mapbase-source/source-sdk-2013/blob/master/sp/src/vscript/vscript.cpp

I will change the contribution guidelines to make the correct formatting more clear. Sorry for the confusion.

**Comment by Blixibon** [comment]
I left this PR behind a while ago and it's not really in a good state to be merged now.

If anyone wants to pick this up again, please open a new branch and put it on the latest version.

**Pull request closed without merging**